docs/config.md

```diff
@@ -25,7 +21,7 @@ To use TypingDNA with WSO2 Identity Server, first you need to configure the auth
     enabled_features=["security.loginVerifyData.typingDNA"]
    ```
 
-#### Deploying TypingDNA Artifacts.
+### Deploying TypingDNA Artifacts
 
 You can either download the TypingDNA artifacts or build the authenticator from the source code.
 
```

Comment:
- To build from the source code:
- To build the authenticator, navigate to the `identity-conditional-auth-typingdna` directory and execute the following command in a command prompt:
- Copy the `org.wso2.carbon.identity.conditional.auth.typingdna.functions-x.x.x.jar` file into the 
   `<IS-Home>/repository/components/dropins` directory and the `api#identity#typingdna#v_.war` file into the `<IS-Home>/repository/deployment/server/webapps` directory.

Reply:
Addressed with https://github.com/wso2-extensions/identity-conditional-auth-typingdna/pull/15/commits/809fb935c083c4e9bed561daf6bd729580a9e074